Tangem/Modules/TokenDetails/TokenDetailsCoordinator.swift

```diff
@@ -47,11 +47,18 @@ class TokenDetailsCoordinator: CoordinatorObject {
             address: options.walletModel.wallet.address,
             amountType: options.walletModel.amountType
         )
-        let swapPairService = SwapPairService(
-            selectedExpressCurrency: options.walletModel.expressCurrency,
-            walletModelsManager: options.cardModel.walletModelsManager,
-            userWalletId: options.cardModel.userWalletId.stringValue
-        )
+
+        let swapPairService: SwapPairService?
+        if !options.walletModel.isCustom {
+            swapPairService = SwapPairService(
+                selectedExpressCurrency: options.walletModel.expressCurrency,
+                walletModelsManager: options.cardModel.walletModelsManager,
+                userWalletId: options.cardModel.userWalletId.stringValue
+            )
+        } else {
+            swapPairService = nil
+        }
```

Comment:
nit: не понимаю что же за привычка такая делать все через отрицание
```suggestion
        if options.walletModel.isCustom {
            swapPairService = nil
        } else {
            swapPairService = SwapPairService(
                selectedExpressCurrency: options.walletModel.expressCurrency,
                walletModelsManager: options.cardModel.walletModelsManager,
                userWalletId: options.cardModel.userWalletId.stringValue
            )
        }
```

Reply:
Не понимаю твоё отрицание отрицаний :D Это просто инструмент

Я приучен к тому, что сначала идет "положительный", "популярный", "правильный" кейс, потом остальное

Reply:
да просто ты же код читаешь и понимаешь о чем он, и не всегда приходишь с пониманием как он должен работать, и сопоставляешь свое понимание с тем как он написан. 
Без отрицаний читается проще)